packages/fabric/src/index.ts

```diff
@@ -0,0 +1,169 @@
+import { Restroom } from "@restroom-mw/core";
+import { ObjectLiteral } from "@restroom-mw/core/src/types";
+import * as crypto from 'crypto';
+// How to include this?
+// import { combineDataKeys } from "@restroom-mw/sawroom/src/lib";
```

Comment:
Guess it's in the wrong place if it's useful to more than one package! Suggest to move it in `@restoroom/utils` and export it from there (and plz refactor also the sawroom package)... probably many other small methods can fit in there also... if you start the way, people will follow your lead 🕺🏻

Reply:
The definition 
```
interface ObjectLiteral {
  [key: string]: any;
}
```
has to be repeated in every package, right?

Reply:
nope we can have a types definition somewhere shared (I can take care of that)!